### ISSUE TITLE
How to finetune the llasa model with nlp description text?

### ISSUE THREAD

**Issue by shuaiyuanxx** [zhenye234/LLaSA_training#48]
Great work, thanks!
I am working on the task of controllable tts and I choose your llasa model as the base model and try to finetune on a 200h speech dataset with nlp description text. 
Now I am facing with a bad performance of the generated speech after finetuning based on llasa-1B model. Before finetuning, I expand the tokenizer with two special tokens: "<|NLP_PROMPT_START|>" and "<|NLP_PROMPT_END|>", and change the chat to : 
chat = [
            {"role": "user", "content": "Convert the text to speech: <|NLP_PROMPT_START|> <|TEXT_UNDERSTANDING_START|>"},
            {"role": "assistant", "content": "<|SPEECH_GENERATION_START|>"}
        ]
I try both lora finetune and finetune methods, but the generated speech performs badly. The pronouciation is not clear and the timbre sounds different with the prompt audio, sometimes the inference outputs unexpected tokens which xcodec2 can't decode to speech.
This is my training loss and eval loss during finetuning:

![Image](https://github.com/user-attachments/assets/79f5421c-37aa-4c0f-a7c4-dd84660b3a33)

This is my training loss and eval loss during finetuning with lora:

![Image](https://github.com/user-attachments/assets/3b09543d-fa04-46ee-b38d-dacb93fe367b)

Please give me some advices on the controllable TTS finetuning task, thanks a lot~

**Comment by zhenye234** [comment]
Hi, I think you can try https://huggingface.co/HKUSTAudio/Llasa-1B-Preserve-TextChat or https://huggingface.co/HKUSTAudio/Llasa-3B-Preserve-TextChat for finetuning since these models preserve the textual instruction following ability from llama-instruct.

**Comment by shuaiyuanxx** [comment]
> Hi, I think you can try https://huggingface.co/HKUSTAudio/Llasa-1B-Preserve-TextChat or https://huggingface.co/HKUSTAudio/Llasa-3B-Preserve-TextChat for finetuning since these models preserve the textual instruction following ability from llama-instruct.

Thanks!!!
